circuit/src/main/kotlin/com/slack/circuit/helpers/ComposableHelpers.kt

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
nit: let's maybe call this file something more specific? `stateUtil.kt` or something